Vagrantfile

```diff
@@ -53,7 +53,10 @@ Vagrant.configure(2) do |config|
 
     config.vm.provision "shell", privileged: false, 
         #args: ["ivi-logging", "https://github.com/Pelagicore/ivi-logging.git", "-DENABLE_DLT_BACKEND=1"],
-        args: ["ivi-logging", "https://github.com/Pelagicore/ivi-logging.git"],
+        args: ["ivi-logging",
+               "https://github.com/Pelagicore/ivi-logging.git",
+               "",
```

Comment:
While we're looking at this, don't we want to set CMAKE_INSTALL_PREFIX=/usr here?